Lib/compositor/font.py

```diff
@@ -143,7 +144,8 @@ def __getitem__(self, name):
     def stringToGlyphNames(self, string):
         glyphNames = []
         for c in string:
-            c = unicode(c)
+            if sys.version_info[0] < 3:
+                c = unicode(c)
```

Comment:
since fontTools is already there just use at the top of the file

```python
from __future__ import unicode_literals
from fontTools.misc.py23 import *
```

its nicer to solve it with imports and not modify the code to much 

thanks!!